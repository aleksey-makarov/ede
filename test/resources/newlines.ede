{ "var": "more" }
---
Some text


More

{{ var }} and ...



