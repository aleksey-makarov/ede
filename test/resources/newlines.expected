Some text


More

more and ...



